spec-main/api-browser-window-spec.ts

```diff
@@ -832,6 +832,22 @@ describe('BrowserWindow module', () => {
       })
     })
 
+    describe('BrowserWindow.getBackgroundColor()', () => {
```

Comment:
We should also test that setting a background color with `setBackGroundColor(color)` results in that color being correctly returned with `getBackgroundColor()`

Reply:
Right! Added test.